packages/tooltip/src/Tooltip.tsx

```diff
@@ -237,9 +253,13 @@ function Tooltip({
   };
 
   useEventListener('click', handleBackdropClick, {
-    enabled: open && triggerEvent === 'click',
+    enabled: enabled && open && triggerEvent === 'click',
   });
 
+  // @ts-ignore
```

Comment:
can we explicitly pull this out of props rather than bringing in lodash/omit

Reply:
I wasn't able to get TypeScript to understand this is a valid prop when trying to destructure it. Maybe if we're ts-ignoring here though, we could just ts-ignore it where we would destructure it...

We have set a precedent of using omit elsewhere though, and lodash is likely to be a dependency of most apps already, so I don't think the cost is all that high here.

Still, if you feel strongly about this, I can change it!

Reply:
I'm trying to remove many usages of lodash/omit with my Box PR, which is why I called this out here. Reasoning makes sense, so I'm ok leaving as-is although my preference would probably just be to push the @ts-ignore to where we're destructuring props

Reply:
I think I actually was able to remove the handling of the testid altogether and handle it through rest. Take a look!

Reply:
👍